### PR TITLE
Request to add brand for cdm4

### PR DESCRIPTION
**Pull request by quantizationbit**
(note this is from Bill at Samsung, please see email sent to mp4reg)

### CODE REVIEW

File: CSV/brands.csv [hunk 1]
```diff
@@ -53,6 +53,7 @@ ccff,Common container file format,DECE
 ceac,CMAF Media Profile - Enhanced AC-3,ETSI AC-3
 cfhd,CMAF Media Profile - AVC HD,CMAF
 cfsd,CMAF Media Profile - AVC SD,CMAF
+cdm4,CMAF Media Profile compatibility to HDR10+ (ST2094-40), SCTE 215-1-1b
```
Comment (cconcolato) [inline CSV/brands.csv:56]: Can you update https://github.com/mp4ra/mp4ra.github.io/blob/dev/CSV/specifications.csv to add SCTE-215-1-1b?

**Comment by quantizationbit** [comment]
Yes, Included SCTE and fixed SMPTE

**Review by quantizationbit — COMMENTED** [review]
Added SCTE 215 as asked.

File: CSV/specifications.csv [hunk 2]
```diff
@@ -91,9 +91,11 @@ QT,QT,"The <a href='http://www.apple.com/quicktime/' target='_blank'>QuickTime</
 RealHD,RealHD,<a href='http://www.realnetworks.com/realmediahd'>RealMedia™ HD</a>
 ross,Ross,<a href='http://www.rossvideo.com/'>Ross Video</a>
 samsung,Samsung,"<a href='http://www.samsung.com/'>Samsung</a> "
+SCTE, SCTE 215-1-1 2020b,"<a href='https://www.scte.org/download-scteisbe-standards/'>SCTE Specifications</a>"
```
Comment (cconcolato) [inline CSV/specifications.csv:94]: The first column is the link name for the specification. It is used in the last column of the brands.csv file. You currently have `SCTE` vs `SCTE 215-1-1b`. You should pick one and we don't allow spaces in link names. Use `SCTE-215-1-1b` in both files.

File: CSV/specifications.csv [hunk 3]
```diff
@@ -91,9 +91,11 @@ QT,QT,"The <a href='http://www.apple.com/quicktime/' target='_blank'>QuickTime</
...
 SDV,SDV,"<a href='http://www.sdcard.org/'>The SD Card Association</a>: SD Memory Card Specifications, Part 8, VIDEO SPECIFICATIONS"
 SL-HDR,SL-HDR,"<a href='http://www.etsi.org/'>ETSI</a> TS 103 433 - High-Performance Single Layer High Dynamic Range System"
 smptevc1,SMPTE,"<a href='http://www.smpte.org/'>SMPTE</a> RP2025:2007, VC-1 Bitstream Storage in the ISO Base Media File Format"
+SMPTE, ST 2094-40:2020, "<a href='https://ieeexplore.ieee.org/document/9095450/'>SMPTE ST 2094-40:2020</a>"
```
Comment (cconcolato) [inline CSV/specifications.csv:98]: This is not referenced directly by a brand and IIUC should be removed.

**Review by cconcolato — APPROVED** [review]

File: CSV/brands.csv [hunk 4]
```diff
@@ -53,7 +53,7 @@ ccff,Common container file format,DECE
 ceac,CMAF Media Profile - Enhanced AC-3,ETSI AC-3
 cfhd,CMAF Media Profile - AVC HD,CMAF
 cfsd,CMAF Media Profile - AVC SD,CMAF
-cdm4,CMAF Media Profile compatibility to HDR10+ (ST2094-40),SCTE-215-1-1b
+cdm4,CMAF Media Profile compatibility to HDR10+ (ST2094-40),SCTE 215-1-1 2020b
```
Comment (dwsinger) [inline CSV/brands.csv:56]: I think that the short name (last column) should not have spaces, as Cyril suggests

File: CSV/specifications.csv [hunk 5]
```diff
@@ -91,7 +91,7 @@ QT,QT,"The <a href='http://www.apple.com/quicktime/' target='_blank'>QuickTime</
 RealHD,RealHD,<a href='http://www.realnetworks.com/realmediahd'>RealMedia™ HD</a>
 ross,Ross,<a href='http://www.rossvideo.com/'>Ross Video</a>
 samsung,Samsung,"<a href='http://www.samsung.com/'>Samsung</a> "
-SCTE-215-1-1b, SCTE 215-1-1 2020b,"<a href='https://www.scte.org/download-scteisbe-standards/'>SCTE Specifications</a>"
+SCTE-215-1-1b,SCTE 215-1-1 2020b,"<a href='https://www.scte.org/download-scteisbe-standards/'>SCTE</a> HEVC Video Constraints for Cable Television Part 1-1 HDR10 Coding"
```
Comment (dwsinger) [inline CSV/specifications.csv:94]: Let's have the same shortname in columns one and two here (and match the suggestion above)?

**Review by dwsinger — CHANGES_REQUESTED** [review]

File: CSV/specifications.csv (hunk 5 above)
Comment (quantizationbit) [reply CSV/specifications.csv:94]: I thought column 2 could be the name of the spec and have spaces (e.g. like other entries such as ISO-CENC, ISO Common Encryption"

File: CSV/brands.csv (hunk 4 above)
Comment (quantizationbit) [reply CSV/brands.csv]: Yes

File: CSV/specifications.csv (hunk 5 above)
Comment (podborski) [reply CSV/specifications.csv:94]: It was intended to be used like that but it looks like the first column is never used and the second one is used to reference the spec. I can fix it and make a new PR for that.

**Pull request merged**